super_csv/serializers.py

```diff
@@ -38,5 +44,22 @@ def get_related_queryset(cls, queryset):
         return queryset.select_related('user')
 
     def get_data(self, operation):
-        data = json.load(operation.data)
-        return CSVOperationDataSerializer(data).data
+        """
+        Get data
+        """
+        try:
+            data = json.load(operation.data)
+        except (FileNotFoundError, ValueError) as error:
+            msg = _('Fail to retrieve file.')
+
+            logger.error('%s\n%s\n%s',
+                         msg, str(error), f'Fail loading file {operation.id} for course {operation.unique_id}.')
```

Comment:
This error is kind of weird and has lots of duplicate info: "Fail to retrieve file\nFile missing\nFail loading file". Trim it down to something like:

```suggestion
            logger.error(f'Error loading operation {operation.id} for course {course_id}: {str(error)}')
```

Reply:
updated the change. I want to preserve some of the translation.